packages/components/src/badge/styles.scss

```
@@ -0,0 +1,38 @@
+@mixin badge-color-variant( $base-color ) {
+	background-color: mix($white, $base-color, 90%);
+	color: mix($black, $base-color, 50%);
+}
+
+$badge-colors: (
+	"info": #3858e9,
+	"warning": $alert-yellow,
+	"error": $alert-red,
+	"success": $alert-green,
+);
+
+.components-badge {
+	background: $gray-100;
+	color: $gray-800;
+	padding: 0 $grid-unit-10;
+	min-height: $grid-unit-30;
+	border-radius: $radius-small;
+	font-size: $font-size-small;
+	font-weight: 400;
+	flex-shrink: 0;
+	line-height: $font-line-height-small;
+	width: fit-content;
+	display: flex;
+	align-items: center;
+	gap: 2px;
+
+	&--has-icon {
+		padding-left: $grid-unit-05;
```

For easier RTL support (we haven't added a lint rule for this yet, sorry).

```suggestion
		padding-inline-start: $grid-unit-05;
```